src/com/etendoerp/sequences/SequenceDatabaseUtils.java

```diff
@@ -86,9 +90,23 @@ public static BaseOBObject getFromClass(Class<? extends BaseOBObject> cls, Seque
     }
 
     public static BaseOBObject searchSequence(Class<? extends BaseOBObject> cls, String whereClause, Map<String, Object> parameters) {
```

Comment:
Bad method name, if this method search sequences then this don't must be return a BaseOBObject

Reply:
I understand the correction. But shouldn't it be out of the scope of this issue?